Sample Data/Custom/ESETInspect_CL.json

```diff
@@ -51,7 +51,7 @@
     "TableName": "PrimaryResult",
     "TenantId": "7bdecb7b-0545-4aa9-bd6b-f7eb05fee857",
     "TimeGenerated": "2020-12-07T10:40:12.774Z",
-    "Type": "ESETEnterpriseInspector_CL",
+    "Type": "ESETInspect_CL",
     "_ResourceId": "",
     "computerId_d": "3",
```

Comment:
Hey @dmaasland, as "computerId_d": "3", is added with "" it acts as string, please add the value as "computerId_d":  3, at all places, thanks.